packages/hash/api/src/graphql/typeDefs/entity.typedef.ts

```diff
@@ -278,6 +281,10 @@ export const entityTypedef = gql`
     Associated Entities to either create/get and link to this entity.
     """
     linkedEntities: [LinkedEntityDefinition!]
+    """
+    Allows UpdatePageContentsActions to reference entities created in other actions. Also allows callers to updatePageContents to find the entity id created for this definition in the result. See UpdatePageContentsResult.
+    """
+    placeholderID: ID
```

Comment:
Seems that `placeholderID` is leaking from Page even though we are only using them when `EntityDefinition`s are used in `UpdatePageAction`. Could this placeholder be moved to the variants on `UpdatePageAction` exclusively and not have this conditional property for all EntityDefinitions?

I guess it would mean having two `placeholderID`s on `InsertBlock` for example, right?

Reply:
👍